### PR TITLE
new tool (TorghostNG) 

### PR DESCRIPTION
**Pull request by D4imyo**

### CODE REVIEW

File: packages/torghostng/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+pkgname=TorghostNG
```
Comment (ikstream) [inline packages/torghostng/PKGBUILD:1]: Only use lower case for pkgname please

```suggestion
pkgname=torghostng
```

**Review by ikstream — CHANGES_REQUESTED** [review]
Please add copying info to the first line as well

```
# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).                                                                                                                                                                                                                                                                                   
# See COPYING for license details.
```

File: packages/torghostng/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+groups=('blackarch' 'blackarch-proxies')
+license=('GPL3')
+depends=('python' 'tor' 'macchanger' 'privoxy' 'curl' 'net-tools')
+makedepends=('git')
+source=('git+https://github.com/GitHackTools/TorghostNG.git')
+sha512sums=('SKIP')
+
+pkgver(){
+  cd TorghostNG
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (noraj) [inline packages/torghostng/PKGBUILD:19]: Replace with `git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'`

File: packages/torghostng/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+package(){
+  cd TorghostNG
```
Comment (noraj) [inline packages/torghostng/PKGBUILD:23]: Create `_pkgname=TorghostNG` and then use `$_pkgname` everywhere

File: packages/torghostng/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  install -dm 755 "$pkgdir/usr/bin/"
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/torghostng/PKGBUILD:29]: [Don't copy the license for GPL](https://wiki.archlinux.org/title/PKGBUILD#license)

File: packages/torghostng/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+  rm *.md LICENSE
+
+  cp -a * "$pkgdir/usr/share/$pkgname"
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+cd /usr/share/$pkgname
+exec python torghostng.py "\$@"
```
Comment (noraj) [inline packages/torghostng/PKGBUILD:37]: use `"$pkgname.py"` instead

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/torghostng/PKGBUILD (hunk 2 above)
Comment (noraj) [reply packages/torghostng/PKGBUILD:19]: Ref. https://github.com/BlackArch/blackarch-pkgbuilds/blob/9e4dffa5e8f5f74374a2a31daf8411a361a51376/PKGBUILD-generic-git#L17-L25

**Comment by noraj** [comment]
@D4imyo Thank you a lot for the time and effort you spent into creating a high quality PKGBUILD. However the BA team decided no to add this tool into BA repository for the [following reasons](https://github.com/BlackArch/blackarch/issues/3125#issuecomment-868549502). I'm sorry this feature request was not triaged earlier. Next time don't hesitate to ask us on discord before starting to work on a PR. I hope you learned a lot about the PKGBUILD process and that this acquired knowledge will help you for a future PR. Thanks again.

**Pull request closed without merging**